biztalk/core/feature-pack-release-definition.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Step 4 - Create release definitions | Microsoft Docs
+description: In Azure DevOps, create a release definition to deploy the BizTalk Server application
+ms.custom: ""
+ms.date: "12/15/2020"
+ms.prod: "biztalk-server"
+ms.reviewer: ""
+
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+author: MandiOhlinger
+ms.author: mandia
+manager: "anneta"
+---
+
+# Step 4: Create release definition
+
+Like build definitions, release definitions are Azure DevOps tasks, and should probably be done by a Azure DevOps admin. As called out in the Step #3, the build definition builds your project within your git repository, and the release definitions deploys it to your BizTalk Server environment. 
+
+
+## Add the release tasks
+
+When the build succeeds, the release definition deploys your application to your BizTalk Server. 
+
+1. Select the **Releases** tab > **New pipeline**. 
+
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-new-pipeline.png" alt-text="Create a new pipeline as a release task for Azure DevOps in the Visual Studio BizTalk Server project.":::
+
+2. Select the **Empty** template > **Apply**:
+
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-empty-release-template.png" alt-text="Add an empty pipeline template as a release task for Azure DevOps in the Visual Studio BizTalk Server project.":::
+
+3. Change the **Environment name** to **Dev**, or whatever you want to call it. 
+
+4. Select **Add artifact**, select your project, your build definition, and select **Add**: 
+
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-add-artifact.png" alt-text="Add an artifact to the pipeline, and choose your project and build definition for Azure DevOps in the Visual Studio BizTalk Server project.":::
+
+5. Go to the **Tasks** tab, add a new task: 
+
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-new-release-tasks.png" alt-text="Add a task to the pipeline release for Azure DevOps in the Visual Studio BizTalk Server project.":::
+
+6. From the list, filter the results, select the **BizTalk Server Application Deployment** task > **Add**:  
+
+    :::image type="content" source="./media/feature-pack-add-build-release-definitions/azure-devops-biztalk-application-deployment-task.png" alt-text="Add a BizTalk Server application eeployment task to the pipeline release for Azure DevOps.":::
+
+    If **BizTalk Server Application Deployment** isn't listed, then install it at [Deploy BizTalk Application](https://marketplace.visualstudio.com/items?itemName=ms-biztalk.deploy-biztalk-application).
+
+7. Select the **Deploy** task, and enter the values:

```

Comment:
Suggestion:
7. Select the **Deploy** task. 
8. Select the **Operation Name** from the list and configure the associated parameters. Your options:
..


and keep the screenshot of the 3 operations here so that you don't have to repeat them on each page